### PR TITLE
Update microsoft-network-server-amount-of-idle-time-required-before-s…

### PR DESCRIPTION
**Pull request by msbemba**
…uspending-session.md

Added more information on what 99999 Minutes represent as it is 8 business hours per day *208 days

Per issue#https://github.com/MicrosoftDocs/windows-itpro-docs/issues/10974

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: windows/security/threat-protection/security-policy-settings/microsoft-network-server-amount-of-idle-time-required-before-suspending-session.md [hunk 1]
```diff
@@ -33,9 +33,9 @@ The **Microsoft network server: Amount of idle time required before suspending s
 
 ### Possible values
 
--   A user-defined number of minutes from 0 through 99,999
+-   A user-defined number of minutes from 0 through 99,999.
 
-    For this policy setting, a value of 0 means to disconnect an idle session as quickly as is reasonably possible. The maximum value is 99999, which is 208 days. In effect, this value disables the policy.
+    For this policy setting, a value of 0 means to disconnect an idle session as quickly as is reasonably possible. The maximum value is 99999(8 business hours per day), which is 208 days . In effect, this value disables the policy.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/security-policy-settings/microsoft-network-server-amount-of-idle-time-required-before-suspending-session.md:38]:
```suggestion
    For this policy setting, a value of 0 means to disconnect an idle session as quickly as is reasonably possible. The maximum value is 99999 (8 business hours per day), which is 208 days. In effect, this value disables the policy.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@vinaypamnani-msft Please merge this PR. Thanks

**Pull request merged**